src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_FOR_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const basePrice = days * PRICE_FOR_DAY;
+
+  if (days >= LONG_TERM) {
+    return basePrice - LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= SHORT_TERM) {
+    return basePrice - SHORT_TERM_DISCOUNT;
+  }
```

Comment:
The logic for applying discounts should be reviewed. Currently, if `days` is greater than or equal to `LONG_TERM`, the function applies the long-term discount and returns immediately, without checking for the short-term discount. If the intention is to apply only one discount, this logic is correct. However, if both discounts should be considered, you might need to adjust the logic to check both conditions before returning.